components/legacy_team_sidebar/legacy_team_sidebar_controller.tsx

```diff
@@ -16,62 +16,70 @@ import * as Utils from 'utils/utils.jsx';
 import SystemPermissionGate from 'components/permissions_gates/system_permission_gate';
 import Pluggable from 'plugins/pluggable';
 
-import TeamButton from './components/team_button.jsx';
+import TeamButton from './components/team_button';
 
-export function renderView(props) {
+type Actions = {
+    getTeams: Function;
+    switchTeam: Function;
+    updateTeamsOrderForUser: Function;
+}
+
+type State = {
+    showOrder: any;
+    teamsOrder: Array<any>;
+}
+
+interface Props extends Pick<any, any>{
+    myTeams: Team[];
+    currentTeamId: string;
+    moreTeamsToJoin: boolean;
+    myTeamMembers: { [key: string]: any };
```

Comment:
Can we specify type here instead of any?

Reply:
yes sorry for that. I think I can fix that with type `Dictionary<TeamMembership>`